.github/actions/js-install/actions.yml

```diff
@@ -0,0 +1,21 @@
+name: js-install
+description: Set up Node.js and pnpm, install dependencies
+
+runs:
+  using: composite
+  steps:
+    - uses: pnpm/action-setup@v2.1.0
+      with:
+        version: "9.0.6"
+
+    - name: Install Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version-file: "js/.nvmrc"
```

Comment:
this should all be `coral_web` instead of `js` right? 😅

Reply:
trueee-- corrected https://github.com/cohere-ai/cohere-toolkit/pull/81/commits/efa9edd8801dbb3b2c861c1d69b56f4effd6089a